.rubocop_todo.yml

```diff
@@ -0,0 +1,67 @@
+# # This configuration was generated by
```

Comment:
Why have you included this file?

Reply:
It was an option to generate a to-do list from the terminal right in the file. I did this before I saw the option to write in the to-do comments in the exact line location. I will remove!